devices/light_1.py

```diff
@@ -0,0 +1,52 @@
+from flask import Flask
+from flask_restful import Resource, Api, reqparse
+import json
+import requests
+
+
+from models.light import Light
+from models.fan import Fan
+HEADERS = {"Content-type": "application/json"}
+app = Flask(__name__)
+api = Api(app)
+
+light_1 = Light('lampa-w-salonie')
+
+
+class Finder(Resource):
+    def get(self):
+        return light_1.id
+
+
+class AddDevice(Resource):

```

Comment:
AddDevice but applies only for light_1? Should be available for multiple types of devices.